src/app/games/teeko/TeekoTutorial.ts

```diff
@@ -0,0 +1,60 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { TeekoState } from './TeekoState';
+import { TeekoDropMove, TeekoTranslateMove } from './TeekoMove';
+import { Coord } from 'src/app/jscaip/Coord';
+import { PlayerOrNone } from 'src/app/jscaip/Player';
+
+const _: PlayerOrNone = PlayerOrNone.NONE;
+const O: PlayerOrNone = PlayerOrNone.ZERO;
+const X: PlayerOrNone = PlayerOrNone.ONE;
+
+export class TeekoTutorial extends Tutorial {
+
+    public tutorial: TutorialStep[] = [
+        // Teeko aim is to align 4 of your piece or draw a shape with it
+        TutorialStep.informational(
+            $localize`Goal of the game`,
+            $localize`The goal of the game is to align your 4 piece, or to make a square with them.`,
+            TeekoState.getInitialState(),
+        ),
+        // The 8 first turn, you drop on empty square
+        TutorialStep.anyMove(
+            $localize`Dropping a piece`,
+            $localize`At your turn, you must drop one piece on any empty space of the board. There is no other restriction.<br/><br/>You're playing Dark, put a piece on the board.`,
+            TeekoState.getInitialState(),
+            TeekoDropMove.from(new Coord(2, 2)).get(),
+            $localize`Congratulations!`,
+        ),
+        // Then the nexts, if nobody won, you move your piece somewhere empty
+        TutorialStep.anyMove(
+            $localize`Moving a piece`,
+            $localize`After you drop all your four pieces, if nobody won yet, you must now move one of your piece to another empty space. The goal remain to be the first to create a line or a square.<br/><br/>You're playing Dark, move a piece.`,
```

Comment:
Once all of your four pieces are placed on the board, you must [...]
J'enleverais "if nobody won yet", là on explique les moves, la condition de victoire elle vient en début + fin du tuto généralement, et c'est common sense que si a un moment la condition de victoire est vérifiée, le jeu s'arrête. (Sinon faut adapter TOUS les tutos pour dire "à votre tour, si personne n'a gagné, ...")

Reply:
la règle "explicite, c'est mieux" à des limites xD

Reply:
le plus explicite c'est leur balancer TeekoRules.ts :D (tpr)